service/src/main/java/bio/terra/tanagra/service/UnderlayService.java

```diff
@@ -1,16 +1,34 @@
 package bio.terra.tanagra.service;
 
+import static bio.terra.tanagra.indexing.job.bigquery.WriteEntityLevelDisplayHints.ENUM_COUNT_ALIAS;
```

Comment:
As a practice this was recommended against in other VWB services - purely coding convention. 
Only downside of wildcard imports is that it might clutter local namespace. 
No concern changing it